libretroshare/src/ft/ftserver.cc

```diff
@@ -1481,16 +1482,18 @@ TurtleSearchRequestId ftServer::turtleSearch(const RsRegularExpression::Lineariz
     return mTurtleRouter->turtleSearch(expr) ;
 }
 
-#warning we should do this here, but for now it is done by turtle router.
+#ifndef RS_NO_WARN_CPP
+#pragma message("csoler 2018-06-10: We should do this here, but for now it is done by turtle router.")
+#endif
 //   // Dont delete the item. The client (p3turtle) is doing it after calling this.
 //   //
-//   void ftServer::receiveSearchResult(RsTurtleSearchResultItem *item)
```

Comment:
How can the compiler generate warnings in a section that is a comment ???

Reply:
This code i commented and could be uncommented. That's why I change it too.